tests/test_writer.py

```diff
@@ -45,14 +45,51 @@ def test_data_container_save_files(self):
         ds_1_xls = pd.read_excel(os.path.join(directory, 'dataset1.xlsx'))
 
         output_dir = os.listdir(directory)
-        print(output_dir)
         rmtree(directory)
         assert sorted(output_dir) == sorted(['aaa.json', 'dataset1.csv', 'dataset1.xlsx'])
 
         assert_frame_equal(container.dataset1, aaa_json)
         assert_frame_equal(container.dataset1, ds_1_csv)
         assert_frame_equal(container.dataset1, ds_1_xls)
 
+    def test_dictionary_save_files(self):
+
+        data_sets = {'dataset1': pd.DataFrame(np.random.normal(size=(100, 2)),
+                                              columns=['A', 'B']),
+                     'dataset2': pd.DataFrame(np.random.normal(size=(120, 3)),
+                                              columns=['A', 'B', 'C'])}
+
+        directory = 'temp_directory'
```

Comment:
Can we rename this into something more random so that we don't accidentally delete an existing `temp_directory` when running this test? This is actually the case for all tests in this script.

Reply:
Sure.